test/unit/vpc.test.ts

```diff
@@ -0,0 +1,75 @@
+import { get } from "lodash";
+import { merge } from "lodash";
+import { baseConfig, pluginConfigExt, runServerless } from "../utils/runServerless";
+import ServerlessError from "../../src/utils/error";
+import {AwsCfInstruction} from "@serverless/typescript";
+
+describe("vpc", () => {
+    it("should have subnets and security groups", async () => {
```

Comment:
If I understand correctly what this test creates, it could be improved to:

```suggestion
    it("should put Lambda functions in the VPC", async () => {
```